src/board/chain/test.rs

```diff
@@ -3,10 +3,13 @@
 use board::Black;
 use board::Chain;
 use board::Coord;
+use std::collections::HashSet;
 
 #[test]
 fn show_returns_a_legible_string_for_the_chain() {
-    let mut c = Chain::new(1, Black, Coord::new(7,7), vec!(Coord::new(1,1)));
+	let mut set = HashSet::new();
```

Comment:
I don't know how that's even possible, eclipse is supposed to replace tab with spaces